pkg/bootstrap/api/types.go

```diff
@@ -46,6 +46,13 @@ const (
 	// other value is assumed to be false. Optional.
 	BootstrapTokenUsageSigningKey = "usage-bootstrap-signing"
 
+	// BootstrapTokenUsageAuthentication signals that this token should be used
+	// as a bearer token to authenticate against the Kubernetes API. The bearer
+	// token takes the form "(token-id):(token-secret)" and authenticates as the
```

Comment:
@ericchiang Wait with changing this until we've gotten it cleared out in a meeting now